src/WorldCup/ScoreBoard.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Cheba\PhpUnit\WorldCup;
+
+class ScoreBoard
+{
+    protected array $games = [];
+    public function startGame($homeTeam, $awayTeam): object
+    {
+        return new Game($homeTeam, $awayTeam);
+    }
+
+    public function updateScore($game, $homeScore, $awayScore): void
+    {
+        $game->homeScore = $homeScore;
+        $game->awayScore = $awayScore;
+    }
+
+    public function finishGame($game): void
+    {
+        $this->games = array_filter($this->games, function ($g) use ($game) {
```

Comment:
Comparison on reference is not production practice. In production, games are stored and reconstructed from another source. 

Please use another method - assign ID to Game or create `equals` method in Game

Reply:
Created equal method in Game.